src/pages/Details/components/Option.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import { AiOutlineClose } from 'react-icons/ai';
+import './Option.scss';
+
+const Option = ({ product }) => {
+  const [productDetails, setproductDeatils] = useState([{}]);
+  const [productTheNumber, setproductTheNumber] = useState(1);
+
+  const incrementCount = e => {
+    e.preventDefault();
+    setproductTheNumber(productTheNumber => productTheNumber + 1);
+    // setproductTheNumber(productTheNumber + 1);
+  };
+  const decrementCount = e => {
+    e.preventDefault();
+    const value = productTheNumber - 1;
+    if (value < 0) return;
+    setproductTheNumber(value);
+  };
+  useEffect(() => {
+    fetch('data/details.json')
```

Comment:
```suggestion
  const incrementCount = e => {
    e.preventDefault();
    setproductTheNumber(productTheNumber => productTheNumber + 1);
    // setproductTheNumber(productTheNumber + 1);
  };
  
  const decrementCount = e => {
    e.preventDefault();
    const value = productTheNumber - 1;
    if (value < 0) return;
    setproductTheNumber(value);
  };
  
  useEffect(() => {
    fetch('data/details.json')
```

함수 끼리도 가독성을 위해 한칸씩 띄워주세요